GameData/RealismOverhaul/RealPlume_Configs/RaiderNick/rn_us_probes.cfg

```diff
@@ -321,3 +321,84 @@
         }
     }
 }
+
+@PART[rn_kh9_ds]:FOR[RealPlume]:NEEDS[SmokeScreen]
+{
+    PLUME
+    {
+        name = Hypergolic-OMS-Red            //pre-fabbed plume you want
+        transformName = thrustTransform //which transform to attach the plume
+        localRotation = 0,0,0           //Optional - Any rotation needed
+        localPosition = 0,0,0.7           //Any offset needed
+        fixedScale = 1.8                  //Size adjustment to resize to engine
+        energy = 1                      //Adjust length of plume
+        speed = 1                       //Adjust speed to fit resize, 
+                                        //generally close to 1:1 with scale.
+    }
+    @MODULE[ModuleEngines*]
+    {
+        @name = ModuleEnginesRF
+    }
+    @MODULE[ModuleEngineConfigs]
+    {
+        %type = ModuleEnginesRF
+        @CONFIG,*   //Add the effect to every engine config
+        {
+            %powerEffectName = Hypergolic-OMS-Red
+        }
+    }
+}
+
+@PART[rn_kh9_campod_srb]:FOR[RealPlume]:NEEDS[SmokeScreen]
+{
+    PLUME
+    {
+        name = Solid-Vacuum            //pre-fabbed plume you want
+        transformName = thrustTransform //which transform to attach the plume
+        localRotation = 0,0,0           //Optional - Any rotation needed
+        localPosition = 0,0,0.47           //Any offset needed
+        fixedScale = 0.3                  //Size adjustment to resize to engine
+        energy = 1                      //Adjust length of plume
+        speed = 1                       //Adjust speed to fit resize, 
+                                        //generally close to 1:1 with scale.
+    }
+    @MODULE[ModuleEngines*]
+    {
+        @name = ModuleEnginesRF
+    }
+    @MODULE[ModuleEngineConfigs]
+    {
+        %type = ModuleEnginesRF
+        @CONFIG,*   //Add the effect to every engine config
+        {
+            %powerEffectName = Solid-Vacuum
+        }
+    }
+}
+
+@PART[rn_kh9_campod_s_srb]:FOR[RealPlume]:NEEDS[SmokeScreen]
+{
+    PLUME
+    {
+        name = Solid-Vacuum            //pre-fabbed plume you want
+        transformName = thrustTransform //which transform to attach the plume
+        localRotation = 0,0,0           //Optional - Any rotation needed
+        localPosition = 0,0,0.39           //Any offset needed
+        fixedScale = 0.15                  //Size adjustment to resize to engine
+        energy = 1                      //Adjust length of plume
+        speed = 1                       //Adjust speed to fit resize, 
+                                        //generally close to 1:1 with scale.
+    }
+    @MODULE[ModuleEngines*]
+    {
+        @name = ModuleEnginesRF
+    }
+    @MODULE[ModuleEngineConfigs]
+    {
+        %type = ModuleEnginesRF
```

Comment:
Move to main part or delete.  (I think there is already a general patch that changed ModuleEngines to ModuleEnginesRF.  Just set the type once in the proper MEC patch, if you're writing your own.)